ci/scripts/github/tests.sh

```
@@ -23,16 +23,17 @@ export REPORTS_DIR=${WORKSPACE_TMP}/reports
 mkdir -p ${REPORTS_DIR}
 get_lfs_files
 
-create_env group:dev extra:all
+create_env
 rapids-logger "Git Version: $(get_git_tag)"
 
 rapids-logger "Running tests with Python version $(python --version) and pytest version $(pytest --version) on $(arch)"
 set +e
-
+``
```

Shellcheck (0.11.0)

[warning] Remove backticks to avoid executing output (or use eval if intentional).

(SC2092)